lib/esbonio/esbonio/lsp/sphinx/__init__.py

```diff
@@ -244,9 +244,8 @@ def initialize(self, params: InitializeParams):
             **typing.cast(Dict, params.initialization_options)
         )
 
-        self.app = self._initialize_sphinx()
-
     def initialized(self, params: InitializedParams):
+        self.app = self._initialize_sphinx()
```

Comment:
I assume moving this is necessary so that VSCode processes the `buildComplete` notificiations?

Reply:
Correct. This line has to be moved here, so that on VSCode side `onReady` executes fully and the hook to `buildComplete` notifications can then take effect.

Without this move, `buildComplete` notifications are ignored (since the hook does not yet take effect).